octoprint/umbrel-app.yml

```diff
@@ -0,0 +1,28 @@
+manifestVersion: 1
+id: octoprint
+category: Networking
```

Comment:
We might need a whole new category now with OctoPrint! Could also fit under the current category "Productivity"